# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1506362185 25200
#      Mon Sep 25 10:56:25 2017 -0700
# Node ID f3b60734ac9b19fc47dda1b78c954b98a9f539ee
# Parent  8f3db1ac787747552083d35ebdc3cf065a10a379
build: clean generated localization files and default-qt-settings

* libgui/module.mk (%canon_reldir%_CLEANFILES): Include $(LOCALES) and
%reldir%/default-qt-settings in the list.
(%canon_reldir%_DISTCLEANFILES): Delete them from the list.

diff --git a/libgui/module.mk b/libgui/module.mk
--- a/libgui/module.mk
+++ b/libgui/module.mk
@@ -131,22 +131,20 @@ DIRSTAMP_FILES += \
   $(TRANSLATIONS) \
   %reldir%/default-qt-settings.in \
   %reldir%/liboctgui-build-info.in.cc \
   %reldir%/mk-default-qt-settings.in.sh
 
 EXTRA_DIST += $(%canon_reldir%_EXTRA_DIST)
 
 %canon_reldir%_CLEANFILES += \
+  $(LOCALES) \
+  %reldir%/default-qt-settings \
   %reldir%/liboctgui-build-info.cc
 
-%canon_reldir%_DISTCLEANFILES += \
-  %reldir%/default-qt-settings \
-  $(LOCALES)
-
 CLEANFILES += $(%canon_reldir%_CLEANFILES)
 DISTCLEANFILES += $(%canon_reldir%_DISTCLEANFILES)
 MAINTAINERCLEANFILES += $(%canon_reldir%_MAINTAINERCLEANFILES)
 
 libgui-clean:
 	rm -f $(%canon_reldir%_CLEANFILES)
 
 libgui-distclean: libgui-clean
